app/popup.tsx

```diff
@@ -44,6 +46,10 @@ export class Popup extends React.Component<POPUP_PROPS & DispatchProps, {}> {
     MessageService.getRequests(this.props.tabId, requests => {
       this.props.updateField("requests", requests);
     });
+    if (!this.isUrlInValid(this.props.tabUrl)) {
+      this.props.updateField("errorMessage", "");

```

Comment:
Tried doing that. `this.props.updateField("errorMessage", "")` doesn't get called on `ComponentWillUnmount`. It may be that the component gets unmounted before the call gets executed. Hence called it on `componentWillMount`

Reply:
its componentWillUnmount. 
Any error its showing?